subiquitycore/tui.py

```diff
@@ -205,7 +208,8 @@ def hide_load():
     async def wait_with_progress(self, awaitable):
         return await self._wait_with_indication(awaitable, self.show_progress)
 
-    async def _move_screen(self, increment, coro):
+    async def _move_screen(self, increment, coro) \
+            -> Optional[Union[BaseView, Callable[[], BaseView]]]:
```

Comment:
This function finishes with a bare `return`, which of course means `return None` but is upsetting to my C-laden eyeballs.   Would you make that explicit?

Reply:
Oh yes, good catch!

Actually this `else` block is unreachable code now ; so I will just get rid of it.
One can't reach the `else` block if there is an unconditional `return` statement in the `try` block.

This `else` block became dead code with this patch: a7bcc7faf081054dc2ec20871c44e952335c50b0

Reply:
even better!